### DISCUSSION TITLE
Unable to import the code from gitlab to github

### DISCUSSION THREAD

**Discussion by Ramadevich** [community/community#31589] [Actions]
Hi Team,

I'm getting error as "There was an error pushing commits to GitHub". I have tried restarting the import process, but facing the same issue.
![error](https://user-images.githubusercontent.com/112399030/187876443-da34432f-bcad-4403-b205-3ed3e5fe68af.png)

**Comment by airtower-luna** [comment]
In that case you should do as it says above the error message and [contact GitHub support](https://support.github.com/request). Us other users can't see into the import system, so we can't help there. Good luck!
